src/main/java/de/srendi/advancedperipherals/client/screens/KeyboardScreen.java

```diff
@@ -0,0 +1,126 @@
+package de.srendi.advancedperipherals.client.screens;
+
+import com.mojang.blaze3d.vertex.PoseStack;
+import dan200.computercraft.client.gui.ClientInputHandler;
+import dan200.computercraft.client.gui.widgets.WidgetTerminal;
+import dan200.computercraft.core.terminal.Terminal;
+import dan200.computercraft.shared.computer.core.InputHandler;
+import de.srendi.advancedperipherals.client.screens.base.BaseScreen;
+import de.srendi.advancedperipherals.common.container.KeyboardContainer;
+import net.minecraft.client.KeyMapping;
+import net.minecraft.client.Minecraft;
+import net.minecraft.network.chat.Component;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.world.entity.player.Inventory;
+import org.jetbrains.annotations.NotNull;
+import org.lwjgl.glfw.GLFW;
+
+/**
+ * A simple screen but without any rendering calls. Used to unlock the mouse so we can freely write stuff
+ * <p>
+ * We just create a terminal which is used to forward all the key presses and mouse clicks but we don't render it.
+ */
+public class KeyboardScreen extends BaseScreen<KeyboardContainer> {
```

Comment:
If you do not extend from `AbstractContainerScreen`, but inherit `Screen` or `GuiComponent`, then jade should not render on it